app/src/main/java/com/inveitix/android/clue/database/DBHelper.java

```diff
@@ -197,37 +233,61 @@ public void close() {
         database.close();
     }
 
+    /**
+     * Method for reading data from shapes table
```

Comment:
Repeats what the name of the method says. Either elaborate or remove documentation.